client/verta/docs/tutorials/deployment/endpoint_update_config.rst

```diff
@@ -0,0 +1,72 @@
+Updating Endpoint With a Configuration File
+===========================================
+
+.. TODO: link to Endpoint.update tutorial
+
+Verta's Client and CLI support updating Endpoint with JSON or YAML configuration file. This tutorial will show how it can be done, using the Client and CLI.
+
+Configuration File
+------------------
+
+Sometimes, it is more convenient to specify update configuration in a separate, human-readable file, than to have everything in a Python script or a split CLI command.
+
+For example:
+
+.. code-block:: json
+
+        {
+            "run_id": "<experiment run id>",
+            "strategy": "canary",
+            "canary_strategy": {
+                "progress_step": 0.05,
+                "progress_interval_seconds": 30,
+                "rules": [
+                    {
+                        "rule": "latency_avg_max",
+                        "rule_parameters": [
+                            {
+                                "name": "threshold",
+                                "value": "0.1"
+                            }
+                        ]
+                    },
+                    {
+                        "rule": "error_4xx_rate",
+                        "rule_parameters": [
+                            {
+                                "name": "threshold",
+                                "value": "1"
+                            }
+                        ]
+                    }
+                ]
+            }
+        }
+
+.. TODO: Link to configuration file fields.
+
+This allows for easier inspection and modification later on. Users can find out more about the fields of the configuration file here.
+
+In the next sections, we will go through how to update the Endpoint with this configuration file, using the Client and the CLI.
+
```

Comment:
```suggestion
```

The good thing about tutorials is that they are short enough that these sort of self-references can be skipped =)

Reply:
Done. Yeah that's kinda redundant.